README.md

```diff
@@ -1,19 +1,19 @@
 # HUB75
 
-![Example image](./example.jpg)
+![Example image](example.jpg)
 
 Library for controlling the cheap RGB matrix displays with the interface colloquially
 known as hub75 with `embedded-graphics` & `embedded-hal` impls in rust.
 
 Currently only supports panels with a resolution of 64x32 (tested on panel "P3-(2121)64*32-16S-D10").
 
 See
-[rpi-rgb-led-matrix](https://github.com/hzeller/rpi-rgb-led-matrix/blob/master/wiring.md)
+(rpi-rgb-led-matrix)[https://github.com/hzeller/rpi-rgb-led-matrix/blob/master/wiring.md]
```

Comment:
Already fixed in git, but not in the current release, this is the wrong way around again

Reply:
Huh, apologies. Was flying through the fixes too fast I think. I've un-done this breaking change.